jdi-light-vuetify/src/main/java/com/epam/jdi/light/vuetify/asserts/ChipGroupAssert.java

```diff
@@ -63,11 +64,4 @@ public ChipGroupAssert column() {
         jdiAssert(element().isColumn() ? "column" : "not column", Matchers.is("column"));
```

Comment:
Please refactor all methods:
- delete part with ternary operator
- add error message for assert

Reply:
done except methods which call ChipAssert